test/test_opsum_to_ttn.jl

```diff
@@ -1,12 +1,31 @@
 using Dictionaries
 using ITensors
+using ITensorGaussianMPS: hopping_hamiltonian
 using ITensorNetworks
 using Random
+using LinearAlgebra: eigvals
 using Test
 
+function _to_matrices_with_same_inds(t::ITensor,s::ITensor)
```

Comment:
I don't see why this function needs to accept two tensors, or why it needs to make the indices match since it just removes the indices when converting to `Matrix` anyway. The version I proposed seems simpler.

Reply:
It doesn't make sure that the indices are in the same order.

Reply:
Ah, since it's combined there's nothing to order, right.